scripts/build-rpm.sh

```diff
@@ -63,16 +70,28 @@ for version in $versions; do(
 	rm -rf $download_dir
 	mkdir -p $download_dir
 
-	git clone https://github.com/drush-ops/drush.git $download_dir
+	if [ $releasenum -le "6" ]
+	then
+		git_dir="https://github.com/pantheon-systems/drush.git"
+	else
+		git_dir="https://github.com/drush-ops/drush.git"
+	fi
+
+	git clone $git_dir $download_dir
 	cd $download_dir
+	if [[ $version == "6.2.0-remove-uname" ]] || [[ $version == "5.10.0-production" ]]
```

Comment:
Now, you can test to see if version_untampered is or is not the same as version.  If we had to convert the version, then checkout $version_untampered.

Actually, I guess we can just always check out $version_untampered here.